packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -1456,15 +1495,16 @@ export const DialogueMatchingPage = ({classes}: {
           <li>You can then see each other's answers, and choose whether start a dialogue</li>
         </ul>
         
+        <CheckNotificationControl classes={classes} />
```

Comment:
Ideally `CheckNotificationControl` would be a registered component that takes `classes` with the `styles` HoC (for now)